en/docs/observe-and-analyze/analyze/choreo-insights-api.md

```diff
@@ -152,6 +152,16 @@ Fetches all the environments of an organization.
 
 The organization filter.
 
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">projectId</td>
+<td valign="top"><a href="#string">String</a></td>
+<td>
+
+The project ID by which the results need to be filtered. It is optional to specify a value for this
```

Comment:
Since we follow the Google style guide and the recommendation is to use the term on-premises [1], shall we change `on-prem` to `on-premises`?
[1] https://developers.google.com/style/word-list#on-premises

Reply:
We use term on-prem key in the Choreo console. So shall we use that here as well?
![Screenshot 2023-04-20 at 16 12 39](https://user-images.githubusercontent.com/19996612/233343017-bdd4adc5-f591-43bf-bc1e-568ae61fdb9d.png)

Reply:
We might have to get that changed. I'll merge this as is for now and let's update after correcting the term in the Choreo Console.